gnat2goto/driver/gnat2goto_itypes.adb

```diff
@@ -96,13 +98,35 @@ package body Gnat2goto_Itypes is
                                                     "Unknown Ekind"));
    end Do_Itype_Definition;
 
+   ----------------------------
+   -- Do_Itype_Array_Type --
+   ----------------------------
+
+   function Do_Itype_Array_Type (E : Entity_Id) return Irep is
+      Var : constant Node_Id :=
+         Object_Definition (Associated_Node_For_Itype (E));
+   begin
+      case Nkind (Var) is
+      when N_Constrained_Array_Definition =>
+         return Do_Constrained_Array_Definition (Var);
+      when N_Unconstrained_Array_Definition =>
+         return Do_Unconstrained_Array_Definition (Var);
+      when others =>
+         return Report_Unhandled_Node_Irep
+           (E, "Do_Itype_Array_Type", "Unknown array type");
```

Comment:
Can you make this Report_Unhandled actually report *which* NKind is not handled? This is important for reviewing the list of unsupported features and making sure all are visible (otherwise different NKinds would get counted as one type)

Reply:
+10 what he said.